Content.Shared/GameObjects/Components/Mobs/State/SharedCriticalMobState.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Content.Shared.Alert;
+using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Shared.GameObjects.Components.Mobs.State
+{
+    /// <summary>
+    ///     A state in which an entity is disabled from acting due to sufficient damage (considered unconscious).
+    /// </summary>
+    public abstract class SharedCriticalMobState : BaseMobState
+    {
+        protected override DamageState DamageState => DamageState.Critical;
+
+        public override void EnterState(IEntity entity)
+        {
+            base.EnterState(entity);
+
+            if (entity.TryGetComponent(out SharedAlertsComponent status))
+            {
+                status.ShowAlert(AlertType.HumanCrit); // TODO: combine humancrit-0 and humancrit-1 into a gif and display it
+            }
```

Comment:
Additionally I don't see any code for falling down here, should be an IActionBlocker that gets added and removed probably